src/main/resources/camunda/claimant_response_spec.bpmn

```diff
@@ -424,7 +424,7 @@
       <bpmn:conditionExpression xsi:type="bpmn:tFormalExpression">${flowState == "MAIN.FULL_ADMIT_PAY_IMMEDIATELY"}</bpmn:conditionExpression>
     </bpmn:sequenceFlow>
     <bpmn:sequenceFlow id="Flow_00f0sau" sourceRef="proceedsInHeritageSystem" targetRef="NotifyRoboticsOnCaseHandedOffline" />
-    <bpmn:sequenceFlow id="Flow_0j3xxwh" sourceRef="ClaimantDefendantAgreedMediationNotifyRespondent" targetRef="NotifyRoboticsOnContinuousFeed" />
+    <bpmn:sequenceFlow id="Flow_0j3xxwh" sourceRef="ClaimantDefendantAgreedMediationNotifyRespondent" targetRef="SendingCsvWhenAgreedMediationSpec" />
```

Comment:
Are you sure we need to rename this like that? are we not sending notification on continouous feed for lr v lr claims anymore? if we are still sending it then perhaps this should be captured here and there should be a flag? if we are still supposed to notify robotics as well a send csv file for lr vs lip then it should be a different camunda event for csv file and a path to it with a flag that we are doing it only for lr vs lip claim

Reply:
It should be adding new event

![image](https://github.com/hmcts/civil-camunda-bpmn-definition/assets/91327278/f0a7f3e3-26dc-4a8a-8962-9898a60d80d3)